index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <section class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
+        </section>
```

Comment:
kindly use a `nav` tag for the outer container of the `hamburger` menu for convention, semantics, and user experience(UX) 😄 

ie 
  >
        <nav class="menu_icon">
                <div></div>
                <div></div>
                <div></div>
        </nav>

Reply:
Good